filebeat/input/filestream/input_test.go

```diff
@@ -0,0 +1,175 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I added this benchmark over these 3 PRs:

* https://github.com/elastic/beats/pull/37317
* https://github.com/elastic/beats/pull/37334
* https://github.com/elastic/beats/pull/37345

It seems like you don't have the latest version here, therefore it's marked as a new file.

I think you should update this file to the latest version from `main` and then add a case for normalisation and compare the benchmark results before and after your changes.